### ISSUE TITLE
output subtitles still out of sync

### ISSUE THREAD

**Issue by ah3243** [smacke/subsync#188]
I tried running this on dune and i think because of the different (fictional languages) it failed giving an offset close to a minute. Adding in an option to select that language it should use for syncronization could be really useful or adding a value for the detection confidence to use.

**Comment by bataroland** [comment]
Yeah, my problem too. I tried to run ffsubsync on my video and ass file, then it given almost 1 minute offset, instead of 1 second offset.
